### DISCUSSION TITLE
Operation the only consumes time

### DISCUSSION THREAD

**Discussion by atistabiur** [frePPLe/frepple#534] [Q&A]
HI how do i setup an operation the only consumes time, example for a manufacturing a PWB:
- populate da PWB (consumes: material + resources in human availability + machine availability + operation time to make the pwb)
- test the PWB (consume: resources in human availability + machine availability + operation time to test the populated pwb manufactured in the previous operation)

**Comment by jdetaeye** [comment]
The simplest approach is to model this as an operation of type "routing", with a suboperation for the "manufacturing" and "test" steps.
Frepple will plan the suboperations in sequence.

The suboperations/steps can consume materials and capacity as you wish.  
A suboperation that doesn't consume material and/or capacity (and just takes some time) is perfectly fine.
